src/ui-stream/feed.directive.js

```diff
@@ -39,14 +39,25 @@ function clbFeed() {
 /**
  * ViewModel of an activity used to render the clb-activity directive
  * @param {object} $log angular injection
+ * @param {object} $scope angular injection
  * @param {object} clbStream DI
  * @param {object} clbUser DI
  */
-function FeedController($log, clbStream, clbUser) {
+function FeedController($log, $scope, clbStream, clbUser) {
   var vm = this;
+  vm.activitiesLimit = 15;
 
   activate();
 
+  $scope.loadMore = function() {
```

Comment:
it would be better not to pollute the $scope but add the functions to the 'vm' object instead